src/style.css

```diff
@@ -1 +1,84 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
```

Comment:
The font-family name 'Avenir' should be enclosed in quotes if it contains spaces or special characters. In this case, it's fine without quotes, but ensure consistency with other font-family names.